vue-frontend/src/components/ApplyForCareer.vue

```diff
@@ -152,7 +152,7 @@
                   type="file"
                   class="form-control custom-file-input"
                   name="resume"
-                  accept="application/pdf"
+                  accept="application/pdf,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,.doc,.docx"
```

Comment:
What is this `application/vnd.openxmlformats-officedocument.wordprocessingml.document` format?Is it necessary?

Reply:
- `application/msword` means `.doc`
- `application/vnd.openxmlformats-officedocument.wordprocessingml.document` means `.docx`

Reply:
Then why we have taken `.doc,.docx` ?